conda-forge.yml

```diff
@@ -8,6 +9,6 @@ github:
   branch_name: main
   tooling_branch_name: main
 provider:
-  linux_aarch64: azure
+  linux_aarch64: default
```

Comment:
Does default still point to Travis? I thought that was completely unreliable?

Reply:
not if the build platform is `linux_64`